scripts/get-external-data.py

```diff
@@ -270,6 +282,8 @@ def main():
                             "ogr2ogr error when loading table {}".format(name))
 
                     this_table.index()
+                    if renderuser is not None:
+                        this_table.grant_access(renderuser)
                     this_table.replace(new_last_modified)
```

Comment:
As discussed earlier on, the idea of this table switch operation is to ensure an atomic table switch once a table has been successfully loaded.

My concern was that the table switch should only be done after the loading process for *all* tables has finished. This way, none of the production tables would be replaced by their temporary table counterpart in case of some failure.

@pnorman remarked, that all tables are being treated independently by design and in practice they're either updated separately or used on different zoom levels. Hence, there's no need to have the table switch for all tables at the very end the process.